Makefile

```diff
@@ -0,0 +1,8 @@
+test:
```

Comment:
Also add some convenience scripts as standard stuff becomes a bit tedious to type. Question @davhin or maybe @benmaier do you know of an equivalent to [the `scripts` section in `yarn` ](https://classic.yarnpkg.com/en/docs/cli/run/) in python land?